dd-sdk-support-fragments/README.md

```diff
@@ -0,0 +1,49 @@
+# Datadog Integration to be able to use the TrackFragmentsAsViewsStrategy for old Android Support packages
+
+## Getting Started 
+
+To include the Datadog Fragments tracking strategy Android Support implementation in your project, 
+simply add the following to your application's `build.gradle` file.
```

Comment:
```suggestion
```suggestion
To include the Datadog's Fragment tracking strategy in your project (based on Android's Support Fragment library), 
simply add the following to your application's `build.gradle` file.

**Note**: if you're using the AndroidX AppCompat library, use the [dd-sdk-androidx-fragment](dd-sdk-androidx-fragment) artifact instead.
```
```